components/equipment/EquipmentSection.tsx

```diff
@@ -16,6 +16,24 @@ export function EquipmentSection({ equipment }: { equipment: Equipment }) {
         </div>
         <div className="col-span-12 text-center sm:col-span-11 sm:text-left">
           <p className="text-2xl mb-2">{equipment.name}</p>
+
+          <div className="mb-2 flex flex-row font-semibold">
+            <div className=" mr-1">Class:</div>
+            {equipment.classTypes.length === 1 ? (
+              <div className="=">{equipment.classTypes[0]}</div>
+            ) : (
+              equipment.classTypes.map((type) => (
+                <Img
+                  src={`/unitTypeFlag/${type}.png`}
+                  className="inline mr-1"
+                  height={28}
+                  width={28}
+                  onError={addDefaultSrc}
+                />
+              ))
+            )}
```

Comment:
Remove the word "class:" and center on mobile

Reply:
Thinking more, isn't it redundent to have this on every item? The items do have categories that I think are consident? I'm fine with merging this after it's centered, something to think about though.